source/change-streams/change-streams.rst

```diff
@@ -347,16 +361,52 @@ Driver API
 
   class ChangeStreamOptions {
     /**
-     * Allowed values: ‘updateLookup’.  When set to ‘updateLookup’, the change notification
-     * for partial updates will include both a delta describing the changes to the document,
-     * as well as a copy of the entire document that was changed from some time after the
-     * change occurred. The default is to not send a value.
-     * For forward compatibility, a driver MUST NOT raise an error when a user provides an
-     * unknown value. The driver relies on the server to validate this option.
+     * Allowed values: 'default', 'updateLookup', 'whenAvailable', 'required'.
+     *
+     * The default is to not send a value, which is equivalent to 'default'. By
```

Comment:
Decided to document "default" for posterity, since we're doing the same for "off" with `fullDocumentBeforeChange`. The important line here is "not to send a value" (from [DRIVERS-2231](https://jira.mongodb.org/browse/DRIVERS-2231)), so this will not impact drivers.